tempesta_fw/http.c

```diff
@@ -414,10 +414,10 @@ tfw_http_conn_msg_free(TfwHttpMsg *hm)
 static inline void
 tfw_http_conn_req_clean(TfwHttpReq *req)
 {
-	spin_lock(&((TfwCliConn *)req->conn)->seq_qlock);
+	spin_lock_bh(&((TfwCliConn *)req->conn)->seq_qlock);
```

Comment:
It seems that there is not necessity to insert `*_bh` spin locks into code that is called from SoftIRQ context only (`tfw_http_conn_req_clean()` and `tfw_http_resp_fwd()`), as SoftIRQ  handlers are serialized on the same CPU and simple `spin_lock` without bottom halves disabling should be enough in these cases.

Reply:
Yes, but I didn't find any call from user context (e.g. for `tfw_http_conn_req_clean()`).
On the other hand it seems that using `*_bh` spin locks does not affect either the performance or the lock contention. So this is just a discussion note.

Reply:
Example of process context calls: `tfw_cfgop_update_sg_srv_list()` -> `tfw_sock_srv_start_srv()` -> `tfw_sock_srv_connect_srv()` -> `tfw_sock_srv_connect_try_later()` -> `tfw_connection_repair()` -> `conn_repair = tfw_http_conn_repair()` -> `tfw_http_conn_shrink_fwdq()` -> `tfw_http_req_zap_error()` -> `tfw_http_error_resp_switch()` -> `tfw_http_send_resp()` -> `tfw_http_resp_build_error()` -> `tfw_http_conn_req_clean()`.

Reply:
Yes, i was wrong here. Missed this path.
Thanks.